app/services/orderService.js

```diff
@@ -26,29 +28,26 @@ class OrderService {
         const ticker = await exchange.fetchTicker(strategy.symbol);
         const price = ticker.ask;
 
-        const markets = await exchange.loadMarkets();
-        const market = markets[strategy.symbol];
-        const precision = market.market.precision.amount;
-        //TODO: make sure to reach the minimum amount and prize
+        //TODO: make sure to reach the minimum amount and prize by const markets = await exchange.loadMarkets()[strategy.symbol];
 
         let amount = 0;
         // Fetch the current price, calculate purchase amount this time by type
-        if (strategy.type === '1') {
-            amount = (strategy.base_limit / price).toFixed(precision);
+        if (strategy.type === STRATEGY_TYPE.NORMAL) {
+            amount = (strategy.base_limit / price).toFixed(6);
         } else {
-            amount = valueAveraging(strategy).toFixed(precision);
+            amount = valueAveraging(strategy).toFixed(6);
             if (amount <= 0) {
                 return false;
             }
         }
 
         const balance = await exchange.fetchBalance();
         // console.log(balance);
-        const in_params = {};
-        const orderRes = exchange.createOrder(strategy.symbol, type, side, amount, price, in_params);
+        const inParams = {};
+        const orderRes = exchange.createOrder(strategy.symbol, type, side, amount, price, inParams);
 
-        const buy_times = strategy.buy_times + 1;
-        const now_buy_times = strategy.now_buy_times + 1;
+        const buyTimes = strategy.buy_times + 1;
+        const nowBuyTimes = strategy.now_buy_times + 1;
         const inOrder = {
```

Comment:
What's the meaning of inOrder, refer to newOrder?

Reply:
> What's the meaning of inOrder, refer to newOrder?

yes, I followed the old code naming